### PR TITLE
Run examples as tests. Would have prevented #266

### PR DESCRIPTION
**Pull request by zeckalpha**
Just a thought.

### CODE REVIEW

File: script/test.py [hunk 1]
```diff
@@ -242,10 +244,15 @@ def run_test(path):
...
     print('')
 
+def run_example(path):
+  return run_test(path, example=True)
 
 for dir in ['core', 'io', 'language', 'limit', 'meta']:
   walk(join(WREN_DIR, 'test', dir), run_test)
 
+walk(join(WREN_DIR, 'example'), run_example)
+walk(join(WREN_DIR, 'example', 'import-module'), run_example)
```
Comment (munificent) [inline script/test.py]: This shouldn't be needed. `walk()` does a recursive walk.

**Comment by munificent** [comment]
Good idea!

File: script/test.py (hunk 1 above)
Comment (zeckalpha) [inline script/test.py]: Hmm. I didn't think it did given this line. https://github.com/munificent/wren/pull/268/files#diff-1d8e229374da562b542ae6b3bbeccc1bR250

I'll check and update as needed.

**Comment by zeckalpha** [comment]
Updated. Cleaned it up a bit.

**Comment by munificent** [comment]
I dig it. Thanks!

**Pull request merged**